drivers/CAN.h

```diff
@@ -111,6 +111,15 @@ class CAN {
      * @endcode
      */
     CAN(PinName rd, PinName td);
+
+    /** Initialize CAN interface and set the frequency
+      *
+      * @param rd the rd pin
+      * @param td the td pin
+      * @param f the bus frequency in hertz
+      */
+    CAN(PinName rd, PinName td, int f);
```

Comment:
there is already a function called frequency. If you agree, I will call it hz as in the frequency function.
Ok for you ?